README.md

```diff
@@ -405,31 +404,14 @@ class Warnings(IntEnum):
 </details>
 
 ## Examples
-
-### JSON example file
-
-  With version v1 the generated JSON files will have following format:
-
-```ticket.json```
-
-```json
-{
-    "issue_key" : "ISSUE-TEST-1043",
-    "project" : "ISSUE-TEST",
-    "duedate" : "08-23-2024",
-    "assignee" : "somebody else",
-}
-```
-
-  In further versions this might change with the possibility for the user to provide file templates on how JSON files shall be read and written
+check out the all the [Examples](./examples) on how to use the pyJiraCli tool.
 
 ## Used Libraries
 
 Used 3rd party libraries which are not part of the standard Python package:
 
 * [jira](https://pypi.org/project/jira/) - Python library for interacting with JIRA via REST APIs - BSD License (BSD-2-Clause).
 * [colorama](https://github.com/tartley/colorama) - ANSI color support - BSD-3 License
-* [cryptography](https://pypi.org/project/cryptography/) - cryptography is a package which provides cryptographic recipes and primitives to Python developers. - Apache Software License, BSD License (Apache-2.0 OR BSD-3-Clause)

```

Comment:
Please also remove it from the pyproject.toml and other files where it may be written as dependency.